### DISCUSSION TITLE
Path to Victory

### DISCUSSION THREAD

**Discussion by mkianpour** [WeAreMahsaAmini/Core#296] [Ideas]
I think we need more and more hope. We should picture our victory. We should repeat this term: Victory. So, we can design a website to focus on that. Some ideas for this website:
- Counter to Victory
- Positive news (such as: https://twitter.com/Omid6887/status/1575507146946580480?s=20&t=PE7g31YuyQ-IobiWzvVATQ)

**Comment by ptkx132** [comment]
I like the idea, and also we should have section to educate people how to defend themself from Zahak!

**Comment by sarma123456** [comment]
One way could be clarifying exact steps to victory. We can provide clear steps and specify progress for each steps. For example in this link, some of these steps are mentioned: https://www.citizenshandbook.org/get_rid_of_a_dictator.html
